### DISCUSSION TITLE
SGD with sfV2 on VSCode

### DISCUSSION THREAD

**Discussion by marcDevOpsTest** [scolladon/sfdx-git-delta#741] [Q&A] [answered]
Hello,
I've been using SGD for a while now and I integrated it to my pipeline and it works perfectly.
However, after migrating my local VSCode to the SF CLI V2, it cannot seem to find the Plugin to install it or use it.

sf sgd:source:delta --to "HEAD" --from "HEAD~1" --output .
 »   Warning: sgd source delta is not a sf command. 

And When i try to install it with this command : "sf plugins:install sfdx-git-delta " 
Installing plugin sfdx-git-delta@latest... failed
 »   Error: sfdx-git-delta does not exist in the registry. 

Not sure what seems to be the issue. Any help?
Thank you!

**Comment by scolladon** [comment]
Hi @marcDevOpsTest !

That is weird, is it the same with the old sfdx command ?
We have a pipeline that test both command to ensure we are compatible and no problem are detected lately by that.
Are you using an npm / yarn proxy/local registry ?

**Reply by marcDevOpsTest** [reply]
Hello @scolladon ! Thank you for your quick reply.
Actually even when using the  SFDX Command, it's giving me the same output 
`sfdx sgd:source:delta --to "HEAD" --from "HEAD~1" --output .
 »   Warning: sgd source delta is not a sf command.er.`
Not using anything in particular, it's pretty much local basic VSCode.

**Reply by scolladon** [reply, marked as answer]
Ok, thanks. 

Maybe you already tried, but have you uninstall and reinstall sf and sgd plugin ?

**Reply by marcDevOpsTest** [reply]
Hello, 
Thank you for that ! Actually I re-installed everything SF with the latest version, and with that I managed to re-install the plugin and it's working now.
Top 👍
